src/test/java/com/znsio/rpap/AppExampleTest.java

```diff
@@ -0,0 +1,64 @@
+package com.znsio.rpap;
+
+
+import com.znsio.rpap.pages.AppExample;
+import com.znsio.rpap.steps.AppSteps;
+import com.znsio.rpap.utils.JsonDataManager;
+import com.znsio.rpap.utils.JsonDataProvider;
+import org.apache.log4j.Logger;
+import org.testng.annotations.*;
+
+import java.lang.reflect.Method;
+
+
+public class AppExampleTest extends AppBaseTest {
+
+    private AppExample appPage;
+    private AppSteps appSteps;
+    private static final Logger LOGGER = Logger.getLogger(AppExampleTest.class.getName());
+
+    @BeforeMethod
+    private void handleCalculatorPopUps() {
+        log("Inside @BeforeMethod of " + AppExampleTest.class.getSimpleName());
+        appPage = page.getClassInstanceApp(AppExample.class);
+        appPage.handlePopupIfPresent();
+        log("Handling calculator popUps");
+        appSteps = new AppSteps(appDriver, appPage, eyesOnApp);
+    }
+
+    @Test(dataProvider = "getFromJson", priority = -3, description = "Perform addition of two no. on calculator",
+            groups = {"visual"})
+    public void AdditionTest(String input1, String operator1, String input2, String operator2, String expectedMsg) {
```

Comment:
Isn't operator2 always equal? We can just create a constant for it in AppSteps instead of reading it via json file

Reply:
done